.github/workflows/auto-pr-reusable.yaml

```diff
@@ -0,0 +1,76 @@
+name: Auto-PR reusable action
+
+on:
+  workflow_call:
+    inputs:
+      project_base_name:
+        required: true
+        type: string
+        description: 'The base name of the projects used in the repository. For example, "ScalarDB" if projects are named "ScalarDB <version>"'
+
+env:
+  TERM: dumb
+jobs:
+  create_backport_pull_requests:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GH_PR_PAT }}
+      # Escape the PR title. See https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable for details
+      PR_TITLE: ${{ github.event.pull_request.title }}
+    steps:
+      - name: Checkout auto-pr scripts
+        uses: actions/checkout@v4
+        with:
+          repository: scalar-labs/actions
+          sparse-checkout: auto-pr-script
+          ref: main
+          path: auto-pr
+
+      - name: Checkout the target repository
+        uses: actions/checkout@v4
+        with:
+          # This is necessary to avoid unexpected auto-merge in git cherry-pick
+          fetch-depth: 0
+          # This is necessary for git-push
+          token: ${{ secrets.GH_PR_PAT  }}
```

Comment:
```suggestion
          token: ${{ secrets.GH_PR_PAT }}
```

Reply:
Revised in https://github.com/scalar-labs/actions/pull/6/commits/6e0a4d635bc91ab8f7402178887322c40dac122c

Reply:
JFYI: https://github.com/scalar-labs/actions/pull/6/commits/3bc68d77d8916e9f0d5a60c5389c1d302145e09b seems correct